### PR TITLE
Added dashboard URLs to list of centralized apps for g5_updatable notification

### PR DESCRIPTION
**Pull request by jonnysamps**

### CODE REVIEW

**Comment by perryqh** [comment]
:+1:

File: lib/g5_heroku_app_name_formatter/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module G5HerokuAppNameFormatter
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```
Comment (bbauer) [inline lib/g5_heroku_app_name_formatter/version.rb]: Totally up for discussion, but I see this as more a minor...not a patch level release?

**Comment by bbauer** [comment]
:+1:

File: lib/g5_heroku_app_name_formatter/version.rb (hunk 1 above)
Comment (jonnysamps) [inline lib/g5_heroku_app_name_formatter/version.rb]: Sounds good to me

On Fri, Aug 7, 2015 at 12:01 PM, Brian Bauer <notifications@github.com>
wrote:

> In lib/g5_heroku_app_name_formatter/version.rb
> <https://github.com/G5/g5_heroku_app_name_formatter/pull/10#discussion_r36550392>
> :
>
> > @@ -1,3 +1,3 @@
> >  module G5HerokuAppNameFormatter
> > -  VERSION = "0.1.0"
> > +  VERSION = "0.1.1"
>
> Totally up for discussion, but I see this as more a minor...not a patch
> level release?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/G5/g5_heroku_app_name_formatter/pull/10/files#r36550392>
> .
>



-- 
Jonathan

**Pull request merged**